runtime/runtime.m

```diff
@@ -1598,7 +1598,7 @@ -(void) xamarinSetGCHandle: (int) gc_handle;
 	}
 }
 
-int
+unsigned long
```

Comment:
Why `long` and not continue using `int`? Not that it matters much, I'm just curious.

Reply:
is the defined value for size_t I think I could change it to use size_t which is probably nicer, what do you think?

Reply:
I don't think it matters much, the current code is fine with me.